examples/classification/classification/evaluator_binary.py

```diff
@@ -0,0 +1,124 @@
+        is_positive_prediction = prediction.score >= threshold
+        is_positive_sample = ground_truth.classification.label == prediction.label
+        return TestSampleMetricsSingleClass(
+            classification=prediction,
+            threshold=threshold,
+            is_correct=is_positive_prediction == is_positive_sample,
+            is_TP=is_positive_sample and is_positive_prediction,
+            is_FP=not is_positive_sample and is_positive_prediction,
+            is_FN=is_positive_sample and not is_positive_prediction,
+            is_TN=not is_positive_sample and not is_positive_prediction,
+        )
+
+    def compute_test_case_metrics(
+        self,
+        ground_truths: List[GroundTruth],
+        metrics_test_samples: List[TestSampleMetricsSingleClass],
+        labels: Optional[List[str]],
+    ) -> TestCaseMetricsSingleClass:
+        n_tp = sum([mts.is_TP for mts in metrics_test_samples])
+        n_fn = sum([mts.is_FN for mts in metrics_test_samples])
+        n_fp = sum([mts.is_FP for mts in metrics_test_samples])
+        n_tn = sum([mts.is_TN for mts in metrics_test_samples])
+
+        return TestCaseMetricsSingleClass(
+            TP=n_tp,
+            FP=n_fp,
+            FN=n_fn,
+            TN=n_tn,
+            Accuracy=compute_accuracy(n_tp, n_fp, n_fn, n_tn),
+            Precision=compute_precision(n_tp, n_fp),
+            Recall=compute_recall(n_tp, n_fn),
+            F1=compute_f1_score(n_tp, n_fp, n_fn),
+            FPR=n_fp / (n_fp + n_tn) if n_fp + n_tn > 0 else 0,
+        )
+
+    def compute_test_case_plots(
+        self,
+        ground_truths: List[GroundTruth],
+        inferences: List[Inference],
+        metrics: List[TestSampleMetricsSingleClass],
+        gt_labels: List[str],
+        confidence_range: Optional[Tuple[float, float, int]],
+    ) -> List[Plot]:
+        plots: List[Plot] = []
+
+        if confidence_range:
+            plots.extend(self._compute_test_case_confidence_histograms(metrics, confidence_range))
+        else:
+            log.warn("skipping test case confidence histograms: unsupported confidence range")
+
+        positive_label = next(iter({inf.inferences[0].label for inf in inferences}))
+        negative_label = next(iter(set(gt_labels) - set(positive_label)))
+
+        plots.append(
+            compute_roc_curves(
+                [gt.classification for gt in ground_truths],
+                [inf.inferences for inf in inferences],
+                labels=[positive_label],
+            ),
+        )
+
+        plots.append(
+            compute_confusion_matrix(
+                [gt.classification.label for gt in ground_truths],
+                [positive_label if metric.is_TP or metric.is_FP else negative_label for metric in metrics],
+                labels=[positive_label, negative_label],
+            ),
+        )
+        plots = list(filter(lambda plot: plot is not None, plots))
+
+        return plots
```

Comment:
Should we add an Accuracy vs. T plot? On the [linked binary classification example](https://trunk.kolena.io/yoohee/results?modelIds=684&workflow=Image+Classification&testSuiteId=1370&evaluators=N4IgjAzAbAHCBcpZ3mADATgCwF8A04EA7GgkjFguhlAbAKxWa0ixRM045A), I find myself wanting to know the optimal threshold, which isn't quite conveyed in any of the existing plots.